forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -113,6 +121,54 @@ fn validate_and_parse_salts<'a>(
     Ok(contract_salt_map)
 }
 
+async fn deploy_new_proxy(
+    pkg: &BuiltPackage,
+    owner_account_address: &mut Bech32Address,
+    impl_contract: &DeployedContract,
+    build_opts: &BuildOpts,
+    command: &cmd::Deploy,
+    salt: Salt,
+    wallet_mode: &WalletSelectionMode,
+) -> Result<DeployedContract> {
+    println!("  {} proxy contract", "Creating".bold().green());
+    let user_addr = if *owner_account_address != Bech32Address::default() {
+        anyhow::Ok(owner_account_address.clone())
+    } else {
+        // Check if the wallet exists and if not create it at the default path.
+        match wallet_mode {
+            WalletSelectionMode::ForcWallet(password) => {
+                let default_path = default_wallet_path();
+                check_and_create_wallet_at_default_path(&default_path)?;
+                let account = first_user_account(&default_wallet_path(), password)?;
+                *owner_account_address = account.clone();
+                Ok(account)
+            }
+            WalletSelectionMode::Manual => {
+                let secret_key =
+                    select_manual_secret_key(command.default_signer, command.signing_key)
+                        .ok_or_else(|| {
+                            anyhow::anyhow!("couldn't resolve the secret key for manual signing")
+                        })?;
+                bech32_from_secret(&secret_key)
+            }
+        }
+    }?;
+    let user_addr_hex: fuels_core::types::Address = user_addr.into();
+    let user_addr = format!("0x{}", user_addr_hex);
+    let pkg_name = pkg.descriptor.manifest_file.project_name();
+    let contract_addr = format!("0x{}", impl_contract.id);
+    let proxy_contract = build_proxy_contract(&user_addr, &contract_addr, pkg_name, build_opts)?;
+    println!("   {} proxy contract", "Deploying".bold().green());
```

Comment:
```suggestion
    info!("   {} proxy contract", "Deploying".bold().green());
```

The indentation here is off as well